flow/scripts/cts.tcl

```diff
@@ -100,12 +100,11 @@ clock_tree_synthesis -root_buf "$::env(CTS_BUF_CELL)" -buf_list "$::env(CTS_BUF_
 
 set_propagated_clock [all_clocks]
 
-report_time "cts-pre-repair"
-
 estimate_parasitics -placement
 set_dont_use $::env(DONT_USE_CELLS)
-repair_clock_nets
+report_time "cts-pre-repair"
 
+repair_clock_nets
 set_placement_padding -global \
```

Comment:
Add another report_time "cts-post-repair" after repair_clock_nets so we can see the effect of repair clearly. Someday when CTS works better I hope there will be no difference between these two.

Reply:
fixed on a8b635f